.gitignore

```diff
@@ -11,6 +11,7 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+Gemfile.lock
 
```

Comment:
Hey Pavel, was there a reason to remove lock file? Gems normally comes with lock file to keep version control on gem dependencies

Reply:
Nice catch! Looks like my knowledge about it is outdated. I thought it should be gitignored for gems.
Just for the reference proving I wasn't right https://bundler.io/v2.4/man/bundle-install.1.html#THE-GEMFILE-LOCK

Thank you Ittsel! I'll return all `Gemfile.lock`s back.